schema.sql

```diff
@@ -0,0 +1,141 @@
+DROP DATABASE IF EXISTS allbirds;
+
+CREATE DATABASE allbirds;
+
+USE allbirds;
+
+CREATE TABLE product_options (
```

Comment:
You're a trooper for hard-coding all of these. Look forward to seeing the seeding function!